pages/services/kafka/2.3.0-1.0.0/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Kafka 2.3.0-1.0.0
+excerpt:
+title: Kafka 2.3.0-1.0.0
+menuWeight: 9
```

Comment:
This should be a lower menu weight than the previous one (2.2.0-1.0.0/index.md).

8 is fine.